### ISSUE TITLE
Journal for 2025-07-06 to 2025-07-12

### ISSUE THREAD

**Issue by beijaflor** [beijaflor/gen-ai-journal#3]
- https://qiita.com/tamura__246/items/bd5d62b1213610618f20
- https://qiita.com/har1101/items/50482e932daabf8121ec
- https://zenn.dev/n0bisuke/books/geminicli-linebot
- https://zenn.dev/arm_techblog/articles/9b2b7bd8cc9ab1
- https://zenn.dev/gatechnologies/articles/5780de81709e97
- https://zenn.dev/solvio/articles/cefce25e803b50
- https://zenn.dev/kako_jun/articles/1d518f47225f42
- https://zenn.dev/mizchi/articles/claude-code-cheatsheet
- https://zenn.dev/smasato/articles/f72b77bead086a
- https://zenn.dev/oikon/articles/0281640eae5aed
- https://zenn.dev/fujihide/articles/d096f376e2479e
- https://zenn.dev/lincwell_inc/articles/0d99371e1ee1a3
- https://zenn.dev/7shi/articles/20250704-structured-output
- https://zenn.dev/mohy_nyapan/articles/133279a79c7dcd
- https://blog.lai.so/cursor-pricing/
- https://maiyama4.hatenablog.com/entry/2025/07/06/133211
- https://future-architect.github.io/articles/20250708a/
- https://note.com/kakowara/n/n46ca355a6aa6
- https://tech.repro.io/entry/2025/07/08/151618
- https://note.com/fladdict/n/n5046f72bdadd
- https://zenn.dev/karaage0703/articles/a2019f9ee283b8
- https://note.com/shotovim/n/ne3a4db9d7c53
- https://blog.ytake.jp.net/entry/2025/07/07/090000
- https://www.wantedly.com/companies/wantedly/post_articles/988360
- https://bufferings.hatenablog.com/entry/2025/07/01/024100
- https://zenn.dev/elyza/articles/3b25b8e44fc280
- https://www.memory-lovers.blog/entry/2025/07/05/092844
- https://diary.hatenablog.jp/entry/2025/07/06/214907
- https://gfx.hatenablog.com/entry/2025/07/06/182751
- https://zenn.dev/danimal141/articles/ea29bb42d75d43
- https://anond.hatelabo.jp/20250705193628
- https://blog.mozilla.ai/wasm-agents-ai-agents-running-in-your-browser/
- https://wasabeef.jp/blog/claude-code-secure-bash
- https://www.elenaverna.com/p/the-rise-of-the-ai-native-employee
- https://uxdesign.cc/how-to-prompt-figma-makes-ai-better-for-product-design-627daf3f4036
- https://uxdesign.cc/good-metaphors-11-commandments-of-ai-ux-figma-shortcuts-671b965a2d39
- https://www.suzukikenichi.com/blog/google-expanding-ai-mode-to-more-users/
- https://www.generalanalysis.com/blog/supabase-mcp-blog
- https://iamcharliegraham.substack.com/p/junior-roles-arent-going-away
- https://github.com/golioth/tinymcp
- https://jamesgrugett.com/p/what-i-learned-building-an-ai-coding
- https://www.thealgorithmicbridge.com/p/im-losing-all-trust-in-the-ai-industry
- https://vercel.com/blog/the-ai-cloud-a-unified-platform-for-ai-workloads
- https://www.holovaty.com/writing/chatgpt-fake-feature/
- https://workos.com/blog/agentic-ai-examples
- https://git.madhouse-project.org/iocaine/iocaine
- https://www.theverge.com/openai/705999/google-windsurf-ceo-openai
- https://www.wheresyoured.at/anthropic-is-bleeding-out/
- https://ddkang.substack.com/p/ai-agent-benchmarks-are-broken
- https://www.politico.com/news/magazine/2025/07/10/musk-grok-hitler-ai-00447055
- https://internetaddictsanonymous.org/internet-and-technology-addiction/signs-of-an-addiction-to-ai/
- https://developers.googleblog.com/en/genai-processors/
- https://bentoml.com/llm/
- https://substack.gauravvohra.com/p/search-is-dead-long-live-llms-winning-in-the-era-of-llms-ai-overviews-and-geo
- https://qiita.com/inukai-masanori/items/657b53a232575ee8c44f
- https://zenn.dev/mkj/articles/efb017f7a440ca
- https://zenn.dev/prgckwb/articles/logits-processor-zoo-explain
- https://zenn.dev/uedakoki/articles/3e035c4e6f78a7
- https://zenn.dev/tacoms/articles/6a717fe95d5199
- https://zenn.dev/akfm/articles/agent-read-the-nextjs-way
- https://zenn.dev/mohy_nyapan/articles/3d20c9979c9b2f
- https://dev.classmethod.jp/articles/claude-code-hooks-basic-usage/
- https://gigazine.net/news/20250711-xai-grok-4/
- https://blog.scottlogic.com/2025/07/09/genai-sustainability-a-review-of-the-2025-numbers.html
- https://secondthoughts.ai/p/ai-coding-slowdown
- https://www.learningfromexamples.com/p/why-why-why-eliza
- https://simonwillison.net/2025/Jul/10/grok-4/
- https://0din.ai/blog/chatgpt-guessing-game-leads-to-users-extracting-free-windows-os-keys-more
- https://www.axios.com/2025/07/10/ai-apple-stock-market
- https://simedw.com/2025/07/10/gemini-bounding-boxes/
- https://rodyne.com/?p=2911
- https://gist.github.com/habonggil/f6130a68bbc4139c8066aa90c14c986f
- https://blog.nishantsoni.com/p/from-ai-to-agents-to-agencies-the
- https://www.rtrvr.ai/blog/on-the-fly-toolgen
- https://efcl.info/2025/07/05/confirm-pam/
- https://iwashi.co/2025/07/06/12-factor-agents
- https://zenn.dev/karaage0703/articles/e68ee90ca00b0d
- https://docs.anthropic.com/ja/docs/claude-code/devcontainer
- https://zenn.dev/sesere/articles/e3d5695e0a7d14
- https://bliki-ja.github.io/LLMsBringNewNatureOfAbstraction
- https://blog.shibayu36.org/entry/2025/07/07/100000
- https://diary.hatenablog.jp/entry/2025/07/06/214907
- https://qiita.com/kanjiro_shima/items/06c1f20eb264a818d938
- https://blog.3qe.us/entry/2025/07/05/001037
- https://zenn.dev/yareyare/articles/99f176a8b1c3a9
- https://president.jp/articles/-/98050
- https://wasabeef.jp/blog/claude-code-secure-bash
- https://developers.freee.co.jp/entry/coding-agent-mechanism
- https://github.com/KentBeck/BPlusTree3/blob/main/rust/docs/CLAUDE.md
- https://qiita.com/takuya77088/items/44a145c9eb3e45383efb
- https://techracho.bpsinc.jp/hachi8833/2025_07_10/151922
- https://baigie.me/nippo/2025/07/10/ai_ma/
- https://martinfowler.com/articles/exploring-gen-ai/i-still-care-about-the-code.html
- https://note.com/biwakonbu/n/n0f1c75418030
- https://meetscareer.tenshoku.mynavi.jp/entry/20250711-hashimoto
- https://zenn.dev/r_kaga/articles/731fe4636289dc
- https://zenn.dev/calloc134/articles/claude_o4mini_search_attempt_failed
- https://zenn.dev/karaage0703/articles/1c84cd87f55ed5
- https://zenn.dev/ueshiii/articles/6b7561f1a1daae
- https://zenn.dev/tommy109/articles/08e42e1a30b494
- https://coliss.com/articles/build-websites/operation/css/ai-generate-for-tailwindcss-code.html
- https://yidingjiang.github.io/blog/post/exploration/
- https://github.com/TecharoHQ/anubis
- http://jjamie.ideasasylum.com/2025/07/01/you%27re-all-cto-now
- https://fi-le.net/baba/
- https://note.com/suthio/n/nb0c1d5cb1aea
- https://firebase.blog/posts/2025/07/supercharge-firebase-studio-with-mcp
- https://mcp-b.ai/
- https://zenn.dev/schroneko/articles/grok-4-overview-and-review
- https://zenn.dev/techlead/articles/2025-07-07-rust-ghost

# Slides

- https://speakerdeck.com/kaelaela/ccc-local-dev
- https://speakerdeck.com/unson/claude-code-ni-puroziekutoguan-li-yarasetamita
- https://speakerdeck.com/rkaga/agentic-coding-starts-with-testing
- https://speakerdeck.com/schroneko/context-engineering
- https://speakerdeck.com/na0/ai-ezientotokao-ezhi-sudetaji-pan
- https://speakerdeck.com/minodriven/ai-good-code-bad-code
- https://speakerdeck.com/masuda220/forkwell_library_fl_100
- https://speakerdeck.com/ra0kley/mcpgahirakudi-li-kong-jian-qing-bao-jie-xi-noke-neng-xing
- https://speakerdeck.com/twada/agentic-software-engineering-findy-2025-07-edition
- https://speakerdeck.com/s4yuba/chu-xue-zhe-demojin-sugudekiru-claude-codenosheng-chan-xing-wo10bei-shang-gerutips
- https://speakerdeck.com/ryuki0947/the-experience-of-having-claude-code-gather-requirements-was-quite-good
- https://speakerdeck.com/bengo4com/20250704-cloudsign-aitool
- https://speakerdeck.com/cybozuinsideout/github-copilot-utilization

# Might Be Hype

- https://qiita.com/EndOfData/items/a16086cab4af175a9e82
- https://zenn.dev/helloworld/articles/d2a102316f23fd
- https://zenn.dev/0k9p1/articles/513b6fe7973e01
- https://qiita.com/Nakamura-Kaito/items/d32b873d39fab94e2c2d

# Better to be Omitted

- https://qiita.com/tatsuya1221/items/a49ea789f671b124ef8b
- https://qiita.com/hirokiii27/items/5fa7bb3c6dd5739d987f
- https://qiita.com/tamtamatsushi0717/items/5959ed2ddfdc7f776653
- https://zenn.dev/mixi/articles/5a13fa0f749cd2
- https://zenn.dev/7shi/articles/20250704-auto-commit
- https://zenn.dev/kikagaku/articles/79368e7981a00f
- https://kaminashi-developer.hatenablog.jp/entry/2025/07/07/093429
- https://zenn.dev/solar/articles/6a2c9ffaf9fe16
- https://forest.watch.impress.co.jp/docs/serial/aistream/2030333.html
- https://qiita.com/kurifumi/items/95caa20379c8961684e8
- https://zenn.dev/dirtyman/articles/5e6445036ac4f6
